dts/bindings/stepper/gpio-stepper-controller.yaml

```diff
@@ -0,0 +1,23 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 Carl Zeiss Meditec AG
+# SPDX-License-Identifier: Apache-2.0
+
+description: GPIO Stepper controller for darlington transistor arrays or dual H-bridge
+
+compatible: "gpio-stepper"
+
+include: stepper-controller.yaml
+
+properties:
```

Comment:
too simple, use child nodes to handle multiple motors with exactly the same GPIO logic – that would save ROM for multiple driver references to exactly the same driver code – see GPIO LED implementation